pkg/deployments/tasks/20230314-batch-relayer-v5/test/contracts/interfaces/ITetuController.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity >=0.7.0 <0.9.0;
+
+interface ITetuController {
+    function changeWhiteListStatus(address[] memory _targets, bool status) external;
+}
```

Comment:
I don't think we need any of these? We never call these interfaces

Reply:
The ABI reference method is sufficient; I can remove all of this